lib/routes/route.ex

```diff
@@ -85,9 +85,16 @@ defmodule Routes.Route do
   @type subway_lines_type :: :orange_line | :red_line | :green_line | :blue_line | :mattapan_line
   @type branch_name :: String.t() | nil
 
+  # Icons we know we have SVGs for, modify if new icons are added/removed
+  # These names are equivalent to the route names from the Massport GTFS
+  @logan_express_icon_names ["BB", "BT", "DV", "FH", "WO"]
+  @massport_icon_names ["11", "22", "33", "44", "55", "66", "77", "88", "99"]
```

Comment:
💡 There is probably a better name for these than `*_icon_names` as they have to represent something more than the fact that we happen to have icons for them. `@logan_express_routes` and `@massport_routes` is probably more domain-driven.

Reply:
I see that these were moved from elsewhere and otherwise unchanged, but if it's not too much effort, I do agree with this comment - these aren't just the icon names - it's a comprehensive list of the routes.

IMO this is non-blocking - happy to see this fixed either in this PR or in a follow-up.

Reply:
FWIW they're not comprehensive. Comprehensive to me would involve having the Massport GTFS routes.txt involved. These lists _are_ noting the ones we have SVG icons created for and do indeed come from a prior change.